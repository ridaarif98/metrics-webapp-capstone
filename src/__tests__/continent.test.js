import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';
import store from '../redux/configureStore';
import DisplayData from '../compoenets/continent';

describe('Test to see that all objects render correctly.', () => {
  test('Check if Continent renders.', () => {
    const tree = renderer
      .create(
        <Router>
          <Provider store={store}>
            <DisplayData />
          </Provider>
        </Router>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
