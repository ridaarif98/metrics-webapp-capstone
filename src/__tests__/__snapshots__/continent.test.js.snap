// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test to see that all objects render correctly. Check if Continent renders. 1`] = `
<div>
  <div
    className="HeadingDiv"
  >
    <img
      alt="Europe"
      src="europe.png"
    />
    <h1>
      <p>
        Europe
      </p>
    </h1>
  </div>
  <div>
    <div
      className="formDiv"
    >
      <h2>
        Stats By Country
      </h2>
      <form>
        <select
          onChange={[Function]}
        >
          <option>
            Seach By Continent
          </option>
          <option
            value="Africa"
          >
            Africa
          </option>
          <option
            value="Europe"
          >
            Europe
          </option>
          <option
            value="Asia"
          >
            Asia
          </option>
          <option
            value="America"
          >
            America
          </option>
        </select>
      </form>
    </div>
    <p
      className="nodata"
    >
      There is no data of this continent
    </p>
  </div>
</div>
`;
